packages/netlify-cms-widget-markdown/src/MarkdownControl/Toolbar.js

```diff
@@ -187,41 +197,33 @@ export default class Toolbar extends React.Component {
             type="quote"
             label="Quote"
             icon="quote"
-            onClick={onBlockClick}
-            isActive={selectionHasBlock}
+            onClick={this.handleBlockClick}
+            isActive={hasBlock('quote')}
             isHidden={this.isHidden('quote')}
             disabled={disabled}
           />
-          <ToolbarButton
-            type="code"
-            label="Code Block"
-            icon="code-block"
-            onClick={onBlockClick}
-            isActive={selectionHasBlock}
-            isHidden={this.isHidden('code-block')}
-            disabled={disabled}
-          />
```

Comment:
Code blocks must now be inserted. Greatly simplifies insertion logic and clarifies difference between inline code and block.